app/src/views/OperationalLearning/index.tsx

```diff
@@ -216,7 +253,7 @@ export function Component() {
     });
 
     const [
-        pendingExport,,
+        pendingExport, ,
```

Comment:
Remove extra comma

Reply:
This additional comma is used to skip 2nd element and de-structure 3rd element, so it should not be removed!